scripts/verify-payouts.py

```diff
@@ -0,0 +1,88 @@
+"""
+A modified version script will be used in gitcoinco/web repository to facilitate match payouts
+using the production data
+"""
+
+import json
+from decimal import Decimal
+from web3 import Web3
+
+# ========================================= CONFIGURATION ==========================================
+
+# Read our expected data
+with open('outputs/payouts.json') as f:
+  expected_payouts = json.load(f)
+
+# Provider URL
+PROVIDER = 'ws://127.0.0.1:8545/'
```

Comment:
How will we set this in Prod?

Reply:
Like this: https://github.com/gitcoinco/web/blob/55e45f8dab043e5d5ef09e0c517e4f837e6899ec/app/grants/models.py#L1639

Reply:
Ahh got it, so this will need to be modified when added to the Gitcoin backend 👌